RELEASE.md

```diff
@@ -39,6 +39,8 @@ This release contains contributions from:
 
 ## Breaking Changes
 
+* All likelihood methods now take an extra `X` parameter. If you have written custom likelihoods or
+  you have custom code calling likelihoods directly you will need to add this extra parameter.
```

Comment:
```suggestion
* All likelihood methods now take an extra `X` argument. If you have written custom likelihoods or
  you have custom code calling likelihoods directly you will need to add this extra argument.
```
Just to avoid confusion with the gpflow Parameter object.

Reply:
> Just to avoid confusion with the gpflow Parameter object.

Sure.